scripts-available/CDB_Conf.sql

```diff
@@ -0,0 +1,58 @@
+-- This will trigger NOTICE if CDB_CONF already exists
+DO LANGUAGE 'plpgsql' $$
+BEGIN
+    CREATE TABLE IF NOT EXISTS cartodb.CDB_CONF ( PARAM TEXT PRIMARY KEY, CONF TEXT NOT NULL );
```

Comment:
cc @lbosque --> Take a look at @rafatower comment on `cartodb.CDB_Conf` table for backups, I don't know if I must change something at the extension or backup will handle this.